en/v3.0.0/resources/example-behaviour.js

```diff
@@ -1,4 +1,47 @@
 ;(function() {
+
+  // show outdated version warning
+  var container = document.getElementsByClassName('container-fluid')[0];
```

Comment:
I see now you are porting over the changes from https://github.com/openlayers/ol3/pull/5332 to old examples.  So you can ignore the comment above.